### DISCUSSION TITLE
How up to date is the Digital Ocean version?

### DISCUSSION THREAD

**Discussion by yengalvez** [mozilla/hubs#5247] [Hubs Cloud Questions]
Good. I was thinking of using the alpha version of Digital Ocean, since the costs of AWS are too high for me. I just wanted to know if the updates are at the same level or if it is stuck on a very old version. Thanks.

**Comment by pattersonbl2** [comment]
We currently don't have plans to work on an alpha version of the digital ocean.  The digital ocean project would be behind the AWS offering.
